tv_apps/tv_shared/components/smart-button/script.js

```diff
@@ -42,6 +42,17 @@ window.SmartButton = (function(win) {
         }
         break;
       case 'transitionend':
+        // fire the event to trigger wave effect
+        // there are many transition properties in focus
+        // we only want to fire one event
+        if (this.classList.length === 2 &&
```

Comment:
This condition is not accurate. Users may add more than 2 class to class list. In this case, the event is broken.